rosdep/base.yaml

```diff
@@ -3665,6 +3665,19 @@ openssl:
   fedora: [openssl]
   gentoo: [dev-libs/openssl]
   ubuntu: [openssl]
+optipng:
+  debian:
```

Comment:
Since these are all the same we shouldn't call out each version.

Note we also have a policy not to call out `sid` if we do have to call out versions.